packages/fx-core/src/component/driver/deploy/azure/azureStorageDeployDriver.ts

```diff
@@ -112,6 +116,9 @@ export class AzureStorageDeployDriverImpl extends AzureDeployImpl {
     );
     const responses = await Promise.all(tasks);
     const errorResponse = responses.find((res) => res.errorCode);
+    if (errorResponse?._response?.status === HttpStatusCode.INTERNAL_SERVER_ERROR) {
+      throw DeployExternalApiCallError.uploadToStorageRemoteError(sourceFolder, errorResponse);
```

Comment:
I also see there is wrapAzureOperation(), why don't use that method here?

Reply:
They don't have the same return value here and can't use the same error-handling method. And this type of error has only occurred 2 times. It's not worth creating a function to handle it.